<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tactile Mouse Calibrator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Calibration</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDoubleSpinBox" name="spinCalibrationWidth">
           <property name="prefix">
            <string>Width: </string>
           </property>
           <property name="suffix">
            <string> cm</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>10.00000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinCalibrationHeight">
           <property name="prefix">
            <string>Height: </string>
           </property>
           <property name="suffix">
            <string> cm</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>10.00000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendCalibration">
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startCalibrationButton">
           <property name="text">
            <string>Start Calibration</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Physics / USB</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QDoubleSpinBox" name="spinMass">
           <property name="prefix">
            <string>Mass: </string>
           </property>
           <property name="suffix">
            <string> kg</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinStiffness">
           <property name="prefix">
            <string>Stiffness: </string>
           </property>
           <property name="suffix">
            <string> N/m</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>500000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>2000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinDamping">
           <property name="prefix">
            <string>Damping: </string>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>50000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>30.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinUpdates">
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string>USB rate: </string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendPhysics">
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Not connected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDt">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:10px;</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Save to EEPROM</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="canvas">
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
